torchvision/prototype/transforms/functional/_deprecated.py

```diff
@@ -0,0 +1,71 @@
+import warnings
+from typing import Any
+
+import PIL.Image
+import torch
+
+from torchvision.prototype import features
+
+from ._meta import convert_color_space, convert_color_space_image_pil
+
+
+# TODO: this was copied from `torchvision.prototype.transforms._utils`. Given that this is not related to pytree / the
+#  Transform object, we should probably move it to `functional._utils`.
+def is_simple_tensor(inpt: Any) -> bool:
+    return isinstance(inpt, torch.Tensor) and not isinstance(inpt, features._Feature)
+
+
+def to_grayscale(inpt: PIL.Image.Image, num_output_channels: int = 1) -> PIL.Image.Image:
```

Comment:
I don't understand, why aren't we calling the legacy kernel from `_F` after throwing the warnings?

Reply:
In the past we opted to not use deprecated behavior. The latest of these was #6454, but #5675 is the same as this PR but for the transforms.

Reply:
As discussed offline, we will just call the legacy kernels.